napari/components/experimental/monitor/wrapper.py

```diff
@@ -13,8 +13,6 @@
 from pathlib import Path
 from typing import Optional
 
-from ...layerlist import LayerList
```

Comment:
This is a good change

Reply:
This is an example where types help keep you honest. They point out circular dependencies where untyped code does not.

Without types if I give you a Foo, you can go to town using it. Only if  you want to create a Foo do you need the import. 

But with types everyone that touches a Foo needs to import it. And that will hard-fail if things are circular. 

I've seen us work around circular imports with "late imports".  Everyone of those is potentially pointing to a potential design problem. Where fixing the root problem actually makes everything better and clearer. So something to look into on a rainy day.